PatternPal/PatternPal/Services/LoggingService.cs

```diff
@@ -154,6 +155,21 @@ private static LogRequest CompileErrorLog(LogEventRequest receivedRequest)
     /// state of the file and the current state of the file was detected.
     /// </summary>
     /// <param name="receivedRequest">The originally received request from the PP extension</param>
+    /// <returns>A LogRequest populated for this specific event</returns>
+    private static LogRequest FileCreateLog(LogEventRequest receivedRequest)
+    {
+        LogRequest sendLog = StandardLog(receivedRequest);
+        sendLog.EventType = LoggingServer.EventType.EvtFileCreate;
+        sendLog.CodeStateSection = receivedRequest.CodeStateSection;
+
+        return sendLog;
```

Comment:
Could you include a populated `ProjectId`-field here? Since a newly created file belongs to one :).

Reply:
I added this field now